examples/vl6180x_simpletest.py

```diff
@@ -17,6 +17,7 @@
 
 # Create sensor instance.
 sensor = adafruit_vl6180x.VL6180X(i2c)
+# sensor = adafruit_vl6180x.VL6180X(i2c) # This would add a +10 millimeter offset to measurements:
```

Comment:
Needs offset parameter.

Reply:
Ah, thank you for catching this, will add ASAP